tka/tailchonk.go

```diff
@@ -297,7 +299,7 @@ func (c *FS) scanHashes(eachHashInfo func(*fsHashInfo)) error {
 		if !prefix.IsDir() {
 			continue
 		}
-		pb, err := hex.DecodeString(prefix.Name())
+		pb, err := base32StdNoPad.DecodeString(prefix.Name())
```

Comment:
Now that the hash is fully in the (file) filename, do we need to decode the directory name at all anymore? And I guess the other assumption (which i have no idea if it holds) is that 2 characters is always a valid base32 symbol?

Reply:
Removed, you're right we don't need it. Indeed, my goal here was to stop thinking of the prefix directory as base32, and just as "a prefix component of the aum hash string."